README.md

```diff
@@ -61,6 +61,8 @@ ShopifyAPI::Context.setup(
 )
 ```
 
+*Important*: If you are using custom apps you will need to use the value of your access_token instead of api_secret_key
```

Comment:
When I was forced to migrate from private to custom apps this weekend, this little booger caused hours of confusion.